.github/actions/setup-project/action.yml

```diff
@@ -25,7 +25,7 @@ runs:
     - uses: actions/setup-node@v4
       name: ⚙️ Setup Node Version
       with:
-        node-version: '20.8.1'
+        node-version: '20.19.0'
```

Comment:
do we need to update the node version in the Docker files?

Reply:
Yes, I did it for uniformity.

Reply:
I don't see the changes in `Dockerfile.base` and the Worker doesn't use the base image